binary-package.md

```diff
@@ -0,0 +1,76 @@
+---
+title: TiDB Installation Packages
+summary: Learn about TiDB installation packages and the specific components included.
+---
+
+# TiDB Installation Packages
+
+Before [deploying TiUP offline](/production-deployment-using-tiup.md#deploy-tiup-offline), navigate to the [official download interface](https://en.pingcap.com/download/) to download the offline mirror packages of the TiDB server.
+
+TiDB provides two binary offline packages: `TiDB-community-server` and `TiDB-community-toolkit`
+
+The `TiDB-community-server` package contains the following components.
+
+| Component | Change History |
+|---|---|
+| tidb-{version}-linux-amd64.tar.gz |  |
+| tikv-{version}-linux-amd64.tar.gz |  |
+| tiflash-{version}-linux-amd64.tar.gz |  |
+| pd-{version}-linux-amd64.tar.gz |  |
+| ctl-{version}-linux-amd64.tar.gz |  |
+| grafana-{version}-linux-amd64.tar.gz |  |
+| alertmanager-{version}-linux-amd64.tar.gz |  |
+| blackbox_exporter-{version}-linux-amd64.tar.gz |  |
+| prometheus-{version}-linux-amd64.tar.gz |  |
+| node_exporter-{version}-linux-amd64.tar.gz |  |
+| tiup-linux-amd64.tar.gz |  |
+| tiup-{version}-linux-amd64.tar.gz |  |
+| local_install.sh |  |
+| cluster-{version}-linux-amd64.tar.gz |  |
+| insight-{version}-linux-amd64.tar.gz |  |
+| diag-{version}-linux-amd64.tar.gz | New in v6.0.0 |
+| influxdb-{version}-linux-amd64.tar.gz |  |
+| playground-{version}-linux-amd64.tar.gz |  |
+
+The `TiDB-community-toolkit` package contains the following components.
+
+| Component | Change History |
+|---|---|
+| tikv-importer-{version}-linux-amd64.tar.gz |  |
+| pd-recover-{version}-linux-amd64.tar.gz |  |
+| etcdctl | New in v6.0.0 |
+| tiup-linux-amd64.tar.gz |  |
+| tiup-{version}-linux-amd64.tar.gz |  |
+| tidb-lightning-{version}-linux-amd64.tar.gz |  |
+| tidb-lightning-ctl |  |
+| dumpling-{version}-linux-amd64.tar.gz |  |
+| cdc-{version}-linux-amd64.tar.gz |  |
+| dm-{version}-linux-amd64.tar.gz |  |
+| dm-worker-{version}-linux-amd64.tar.gz |  |
+| dm-master-{version}-linux-amd64.tar.gz |  |
+| dmctl-{version}-linux-amd64.tar.gz |  |
+| br-{version}-linux-amd64.tar.gz |  |
+| spark-{version}-any-any.tar.gz |  |
+| tispark-{version}-any-any.tar.gz |  |
+| package-{version}-linux-amd64.tar.gz |  |
+| bench-{version}-linux-amd64.tar.gz |  |
+| errdoc-{version}-linux-amd64.tar.gz |  |
+| dba-{version}-linux-amd64.tar.gz |  |
+| PCC-{version}-linux-amd64.tar.gz |  |
+| pump-{version}-linux-amd64.tar.gz |  |
+| drainer-{version}-linux-amd64.tar.gz |  |
+| binlogctl | New in v6.0.0 |
+| sync_diff_inspector |  |
+| reparo |  |
+| arbiter |  |
+| mydumper | New in v6.0.0 |
+| server-{version}-linux-amd64.tar.gz | New in v6.2.0 |
+| grafana-{version}-linux-amd64.tar.gz | New in v6.2.0 |
+| alertmanager-{version}-linux-amd64.tar.gz | New in v6.2.0 |
+| prometheus-{version}-linux-amd64.tar.gz | New in v6.2.0 |
+| blackbox_exporter-{version}-linux-amd64.tar.gz | New in v6.2.0  |
+| node_exporter-{version}-linux-amd64.tar.gz | New in v6.2.0  |
```

Comment:
These items are already listed in TiDB-community-server table. Please double check.